crates/polars-io/src/parquet/read/read_impl.rs

```diff
@@ -959,9 +959,9 @@ impl FetchRowGroupsFromMmapReader {
 
     fn fetch_row_groups(&mut self, _row_groups: Range<usize>) -> PolarsResult<ColumnStore> {
         // @TODO: we can something smarter here with mmap
-        Ok(mmap::ColumnStore::Local(MemSlice::from_vec(
-            self.0.deref().to_vec(),
-        )))
+        Ok(mmap::ColumnStore::Local(MemSlice::from_static(unsafe {
+            std::mem::transmute(self.0.deref())
```

Comment:
Do we even need the transmute? The`ReaderBytes` is static?

Reply:
ai... I'm starting to remember the situation with ReaderBytes 😄